style.css

```diff
@@ -0,0 +1,35 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  padding: 0 24px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 2em;
+}
+
+#book-title {
+  padding: 0 10px 0 0;
+}
+
+#books-list {
+  display: flex;
+  flex-direction: column;
+  margin-left: -6em;
+}
+
+tr {
+  display: flex;
+  flex-direction: column;
+  font-size: 20px;
+}
+
+button {
+  padding: 3px 10px;
+  margin-bottom: 1em;
+}
```

Comment:
Your project is refactored correctly and works fine as expected, but I think you're missing out on the styling.

- [ ] For this project, alongside the refactor, you're required to style your app to look similar to the wireframe given.

**See the requirements**

<img width="870" alt="Screenshot 2023-01-13 at 9 22 34 AM" src="https://user-images.githubusercontent.com/50941074/212285346-df440a6a-5742-4b1d-8017-ffc7a437d7c0.png">

**Here is how your app looks**

<img width="558" alt="Screenshot 2023-01-13 at 9 23 48 AM" src="https://user-images.githubusercontent.com/50941074/212285496-95046dfa-93ca-4b42-926b-596aa8e48e38.png">

- [ ] Kindly work on your styles for your project to be approved 👍🏽